### ISSUE TITLE
Bulk Action not working

### ISSUE THREAD

**Issue by cortexrd** [cortexrd/Knack-Toolkit-Library#298]
Ok... 
But still not working. 
I added it in the API. 
I added a user role. Assigned myself. Added the role to the view login. Refreshed but nope.

_Originally posted by @Ampurcell in https://github.com/cortexrd/Knack-Toolkit-Library/discussions/47#discussioncomment-9779086_

**Comment by cortexrd** [comment]
@Ampucell,

I've created an issue so we can have a follow-up on your use case.

Is it possible for you to recreate that problem in a temporary debug page for me to test it?

Just add a dummy Yes/No field in a table somewhere and a grid with two action links:  one that sets the flag to Yes and the other to No.  Add that flag in the grid so I can assess the result.

BTW, I tried that password you gave me a while ago, but it's not valid anymore, so I'd need a new one.

Norm
